compiler/tflchef/core/src/ModelChef.cpp

```diff
@@ -552,7 +566,7 @@ GeneratedModel cook(const ::tflchef::ModelRecipe &model_recipe)
   std::vector<flatbuffers::Offset<::tflite::Buffer>> buffer_vec;
 
   // Operation-related
-  std::vector<flatbuffers::Offset<::tflite::OperatorCode>> code_vec;
+  std::vector<flatbuffers::Offset<::tflite::OperatorCode>> operator_code_vec;
```

Comment:
can you explain why we need to rename this?

Reply:
@seanshpark It is not necessary but I just wanted to make users easy to differentiate `operator_code_vec` from `custom_code_vec`.

Reply:
Um, then maybe in another PR.